src/crimson/osd/pg_backend.h

```diff
@@ -190,6 +192,14 @@ class PGBackend
   get_attr_ierrorator::future<> get_xattrs(
     const ObjectState& os,
     OSDOp& osd_op) const;
+  using cmp_xattr_errorator = crimson::os::FuturizedStore::get_attr_errorator;
```

Comment:
might want to be consistent on adding `::` prefix before `crimson` naming space.